src/ui/report.ui

```diff
@@ -0,0 +1,721 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <template class="Report" parent="GtkBox">
```

Comment:
There should be title label Bug report or similar like on other setting pages

Reply:
Title is added